.editorconfig

```diff
@@ -50,9 +50,9 @@ dotnet_style_explicit_tuple_names = true:suggestion
 # CSharp code style settings:
 [*.cs]
 # Prefer "var" everywhere
-csharp_style_var_for_built_in_types = true:suggestion
+csharp_style_var_for_built_in_types = false:none
```

Comment:
I think we at least want suggestion here.  Probably a longer discussion about the level of enforcement. Warning could work as well

Reply:
Yeah I was confused how this worked - I didn't realize saying false here actually means the inverse and hence enforce it.

Reply:
I think the wording I chose here has confused a lot of people.  How would you expect this to work for enforcing this option?  What words would make sense to you?

Reply:
No the naming had nothing to do it with - I just have too much history and remember the old incarnation of this where when you switched off the option in Options and it didn't enforce the reverse.

Reply:
I tried "warning" it was much too aggressive, especially given how much code was already written like this.